app/api/utility.py

```diff
@@ -2,6 +2,7 @@
 
 import json
 import os
+import textwrap
```

Comment:
great example of why learning / reviewing the standard lib is important! didn't know this existed. Very nice